batch_report_pdf_zip/controllers/main.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2024 Quartile (https://www.quartile.co)
```

Comment:
@AungKoKoLin1997 Are you sure that we can claim the full authorship of this file?

Reply:
Yes. I am because I just take the reference from odoo source and the idea comes from me.